src/Native/System.Net.Security.Native/CMakeLists.txt

```diff
@@ -0,0 +1,32 @@
+project(System.Net.Security.Native)
+
+set(CMAKE_INCLUDE_CURRENT_DIR ON)
+
+add_definitions(-DPIC=1)
+
+if (APPLE)
```

Comment:
@bartonjs, if @socket agrees, we would like to defer the CMake tests recommended by to that issue as well